core/src/main/java/org/radargun/Master.java

```diff
@@ -218,6 +218,11 @@ private int executeStage(Configuration configuration, Cluster cluster, int stage
       }
       extras.put(Properties.PROPERTY_SLAVE_INDEX, "-1");
       extras.put(Properties.PROPERTY_PROCESS_ID, String.valueOf(Utils.getProcessID()));
+      configuration.getSetups().forEach(setup ->
```

Comment:
I don't understand this change: envs should be referenced using `${env.MY_VAR}`

Reply:
Removed the commit.
We can do the following:
```xml
....
            <environment>
               <var name="CYCLE_TIME" value="1s" />
            </environment>
.....
cycle-time="${env.CYCLE_TIME:0}"
```
Master will compile and set the value equals 0
When master starts the slave, it will send 1s as an environment variable